tests/support/server.tcl

```diff
@@ -300,7 +300,7 @@ proc wait_server_started {config_file stdout pid} {
     set maxiter [expr {120*1000/$checkperiod}] ; # Wait up to 2 minutes.
     set port_busy 0
     while 1 {
-        if {[regexp -- " PID: $pid" [exec cat $stdout]]} {
+        if {[regexp -- " PID: $pid.*Server initialized" [exec cat $stdout]]} {
```

Comment:
i did this change in order to solve the startup race (because we not init the listeners after printing the PID message), but it got broken when you moved the init back to before the "Server initialized" print.
i'll sort it out (move the listeners init back to before the print), and push soon